### ISSUE TITLE
Fix Request: AvoidParallelFlux missed case

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#516]
Missed when no arguments to parallel():
```
private Flux<Response> processRequests(List<Request> requests, Function<Request, Response> processor) {
        return Flux.fromIterable(requests)
                .parallel() // <- false positive
                .runOn(schedulerDispatcher)
                ....
```
